Tests/Unit/MSFT_xIISLogging.tests.ps1

```diff
@@ -0,0 +1,597 @@
+$global:DSCModuleName = 'xWebAdministration'
+$global:DSCResourceName = 'MSFT_xIisLogging'
+
+#region HEADER
```

Comment:
Can you change this over to use the latest version Test Template header from [here](https://github.com/PowerShell/DscResources/blob/master/Tests.Template/unit_template.ps1#L20)? It may already be the latest version, but if so, just add in the version number like in the template header so we can more easily track which version is in use.